include/mbedtls/build_info.h

```diff
@@ -62,6 +62,11 @@
 #define MBEDTLS_ARCH_IS_X86
 #endif
 
+#if !defined(MBEDTLS_PLATFORM_IS_WINDOWS_ON_ARM64) && \
+    (defined(_M_ARM64) || defined(_M_ARM64EC))
+#define MBEDTLS_PLATFORM_IS_WINDOWS_ON_ARM64
+#endif
```

Comment:
For the time being, it is Okay. 

But I think we should introduce target os macros in future. after that it should be 
```suggestion
#if !defined(MBEDTLS_PLATFORM_IS_WINDOWS_ON_ARM64) && \
    (defined(MBEDTLS_ARCH_IS_ARM64) && defined(MBEDTLS_OS_IS_WINDOWS))
#define MBEDTLS_PLATFORM_IS_WINDOWS_ON_ARM64
#endif
```